extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -206,6 +207,11 @@ export class AmpAdXOriginIframeHandler {
       this.sendEmbedInfo_(this.baseInstance_.isInViewport());
     }));
 
+    this.unlisteners_.push(listenFor(this.iframe, MessageType.USER_ERROR,
+        data => {
+          this.userErrorForAnalytics_(data['message']);
+        }, true, true));
```

Comment:
please see my previous comments. I really don't think we should listen to message from nested iframe here. Also i don't think nested iframe will send us error message.

Reply:
I thought when we talked offline you said you'd prefer to send message around instead of error. What's your suggestion to improve this?

Reply:
Please see. https://github.com/ampproject/amphtml/pull/11002#discussion_r135599948
Let me know if there's any question